cmp/leftRightChooser/impl/ChooserToolbar.js

```diff
@@ -13,6 +13,9 @@ import {Icon} from 'hoist/icon';
 
 /**
  * A Toolbar for the LeftRightChooser.
+ *
+ * This is an implementation class private to Hoist
+ * @private
```

Comment:
I would be fine with just the first line and the annotation (and no "this is an impl..." boilerplate)

Reply:
totally agree with this.  Need to keep these private docs boilerplate to a minimum.